src/app/concern/file-uploader/file-uploader.component.ts

```diff
@@ -38,31 +46,46 @@ export class FileUploaderComponent implements OnInit {
     this.dropArea.nativeElement.classList.remove("highlight");
   }
 
-  public handleDrop(e: any): void {
-    // const dt = e.dataTransfer;
-    // this.uploadedFiles = dt.files;
-    this.upload();
+  public handleDrop($event: DragEvent): void {
+    const itemList: DataTransferItem[] = Array.from($event.dataTransfer.items);
+    const filesAllowed: string[] = this.acceptedFileTypes.split(",");
+    const droppedFiles: File[] = [];
+    const invalidFiles: string[] = [];
+
+    if (itemList.length) {
```

Comment:
Wouldn't this throw an error if itemList is null or undefined?

Reply:
good spot, have updated pr